.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+| WARNING: Please, read carefully before submitting a pull request |
+|------------------------------------------------------------------|
+
+Although this software is open source and we welcome contributions,
+we believe these contributions should be preceded by an open discussion.
+Open discussions tend to result in better solutions to any given problem,
+and help maintain and improve the quality of the software.
+If you would like to see a bug fixed or a new feature implemented,
+please open an issue for the discussion rather than directly opening
+a pull request.
+
+If you would like to contribute please read [CONTRIBUTING.md](/CONTRIBUTING.md).
```

Comment:
This resolves to https://github.com/CONTRIBUTING.md, also incorrect.

You can test this here: https://github.com/cartesi/machine-emulator-rom/issues/new
```
[CONTRIBUTING.md](/CONTRIBUTING.md)
```

and clicking preview.